docs/metrics-guide/averaging-methods.md

```diff
@@ -0,0 +1,60 @@
+# Averaging Methods
```

Comment:
Can we include "macro" and "micro" in the title? It wasn't clear what this was to me at first glance. Maybe:

```
Macro and Micro Averaging
```